package.json

```diff
@@ -89,8 +89,6 @@
   },
   "devDependencies": {
     "@types/bech32": "^1.1.1",
-    "@types/bip32": "^1.0.2",
```

Comment:
how come these are not needed as dev dependencies anymore?

Reply:
look https://www.npmjs.com/package/@types/bip32

Reply:
say no more!